cypress/integration/page.spec.js

```diff
@@ -111,7 +111,7 @@ Cypress.on('fail', (e) => {
   throw e;
 });
 
-describe.skip('', () => {
+describe('', () => {
   beforeEach(() => {
     // if (failed) Cypress.runner.stop();
```

Comment:
Why did you comment on this?

Reply:
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/67290782/6454a374-d153-4112-a813-a155784f2d45)

That is exist in sourse code what i clone from git of mate. About that nothing writed on page of projectm so i don't change that.